### DISCUSSION TITLE
Unable to modify listing columns in 4.0.0 demo site

### DISCUSSION THREAD

**Discussion by candlerb** [netbox-community/netbox#15989] [General]
Raising this as a discussion in case this is a client-side problem (caching, Javascript etc) and to see if anyone else observes similar behaviour.

* Go to https://demo.netbox.dev/dcim/interfaces/
* Click on "Configure table"
* Click on "IP Addresses"
* Click "Add"

Nothing happens - i.e. IP Addresses does not get moved to the right-hand column.

When the dialog is still open, I can click Reset. The box closes, although the main screen remains greyed out. Then I refresh the entire page, then click Configure, and then I *can* add that column and Save.

Subsequently I removed it - this also required a manual page refresh.

Trying a different table, e.g.
* https://demo.netbox.dev/dcim/modules/
then that table's "Add" button also doesn't work.

And if I navigate back to Interfaces, again I can't re-add the IP Addresses column.

**Comment by jeremystretch** [comment]
Seems to be working normally for me. Tested in both Firefox and Chrome.

**Comment by bradmoutoux** [comment]
I am getting this as well on my local install.  I have tried it in Edge, Firefox, and Vivaldi and no matter what I have to manually hit refresh in order to actually add a column to the list. This also goes for running any scripts.  The script runs and the window just sits blank until I manually hit refresh.  Then it populates.  This also seems to happen when exporting data.  So it has to be a common function.  Anyone else have an idea of what this could be?

EDIT: I tried the demo and it works in there.  So it has to be something that isn't working properly, but I do not know what the common function is.

EDIT2: I found my issue.  But not the cause.  The issue was my user account.  No one ele had the issue.  So I went in and deleted my user account.  When i signed back in a new account was created and everything works.
